apps/admin_api/lib/admin_api/v1/plugs/client_auth_plug.ex

```diff
@@ -12,20 +28,21 @@ defmodule AdminAPI.V1.ClientAuthPlug do
   def init(opts), do: opts
 
   def call(conn, opts) do
-    auth =
-      Keyword.get(opts, :enable_client_auth, Application.get_env(:admin_api, :enable_client_auth))
-
-    case auth do
-      "true" ->
+    case enable_client_auth?(opts) do
+      true ->
         conn
         |> parse_header()
         |> authenticate()
 
       _ ->
-        assign(conn, :authenticated, :client)
+        assign(conn, :authenticated, false)
```

Comment:
Are we sure this doesn't break anything anywhere? Worried we're checking the `authenticated` somewhere and expect it to be true :o